src/app/portlets/dot-starter/dot-starter.component.html

```diff
@@ -0,0 +1,240 @@
+<div class="p-grid dot-starter-top-content">
+    <div class="p-col-12">
+        <h2 class="dot-starter-title">{{ 'starter.title' | dm }}</h2>
+        <p class="dot-starter-description" [innerHTML]="'starter.description' | dm: [username]"></p>
+    </div>
+    <div class="p-col-12">
+        <div class="p-grid">
+            <div class="p-md-12 p-lg-7 p-grid dot-starter-top-main__section">
```

Comment:
```suggestion
            <div class="p-md-12 p-lg-7 p-grid">
                <div class="dot-starter-top-main__section">
```

You should not change the grid css, the grid is just the space and your "other" html is to fill that, your other HTML should indeed be response to adjust to the grid.